tests/scripts/thread-cert/sniffer.py

```diff
@@ -102,12 +102,12 @@ def start(self):
     def stop(self):
         """ Stop sniffing. """
 
+        self._transport.close()
+
         self._thread_alive.clear()
```

Comment:
I think we need to clear this first. Otherwise, in `_sniffer_main_loop` it will go right back into the `while` loop. Which on Windows might result in a crash.